docs/guide/http_cache.md

```diff
@@ -390,84 +390,67 @@ ezpublish:
     php app/console cache:clear -vv --env=prod;
     ```
 
-    #### Configuration on platform.sh
+    #### Configuration on Platform.sh
```

Comment:
Below this line, we should maybe explain that these instructions are for a enterprise project, like :

"This chapter explains how to configure Fastly on a Enterprise project on Platform.sh"